app/models/orcid_source_record.rb

```diff
@@ -3,5 +3,5 @@
 class OrcidSourceRecord < ApplicationRecord
   belongs_to :publication, inverse_of: :orcid_source_record, optional: true
 
-  serialize :source_data, JSON
+  serialize :source_data, coder: JSON
 end
```

Comment:
fixes deprecation warnings like this: `DEPRECATION WARNING: Passing the class as positional argument is deprecated and will be removed in Rails 7.2.`